src/WebJobs.Script.WebHost/Management/LinuxSpecialization/PackageDownloadHandler.cs

```diff
@@ -214,5 +229,57 @@ private async Task<bool> IsResourceAccessibleWithoutAuthorization(string resourc
                 return false;
             }
         }
+
+        private string CopyPackageFile(IFileSystem fileSystem = null)
+        {
+            fileSystem ??= FileUtility.Instance;
+            var packageFolderPath = _environment.GetSitePackagesPath();
+
+            if (!fileSystem.Directory.Exists(packageFolderPath))
+            {
+                CopyPackageFileFailed($"{nameof(CopyPackageFile)} failed. SitePackages folder in the data folder doesn't exist.");
```

Comment:
Similar comment here about using `SitePackagesFolderName` constant instead of `SitePackages`

Reply:
I have changed as suggested.